Mesh_3/include/CGAL/Mesh_3/Protect_edges_sizing_field.h

```diff
@@ -140,6 +140,7 @@ class Protect_edges_sizing_field
                              const MeshDomain& domain,
                              SizingFunction size=SizingFunction(),
                              const FT minimal_size = FT(),
+                             DistanceFunction edge_distance = DistanceFunction(),
```

Comment:
@sloriot can you please tell us in which testsuite is this error/warning?

Reply:
I prepared the testsuite yesterday and got it locally. If you don't have it then it was probably a bad merge conflict.

Reply:
> `test_meshing_polyhedron_with_features.cpp` triggers
> 
> ```
>   143 |                              DistanceFunction edge_distance = DistanceFunction(),
>       |                                                               ^~~~~~~~~~~~~~~~~~
> ```

At least do you remember the error message? Because what you quoted is incomplete.

Reply:
DistanceFunction is not default constructible AFAIR

Reply:
@ange-clement can you please compile all Mesh_3 examples and tests to check if all criteria combinations still compile?